extensions/wikia/Follow/FollowHelper.class.php

```diff
@@ -129,24 +77,104 @@ public static function emailNotification( $childTitle, $list, $namespace, $user,
 				$message,
 				$action
 			);
-			$oTask->queue();
+
+			$task->queue();
 		}
+	}
 
-		wfProfileOut( __METHOD__ );
+	/**
+	 * Create and return arrays of at most MAX_WATCHERS_PER_JOB watchers
+	 *
+	 * @param WikiaSQL $watcherSQL
+	 *
+	 * @return array
+	 */
+	private static function getWatcherSets( WikiaSQL $watcherSQL ) {
+		$currentPage = 0;
+		$watcherCount = 0;
+
+		$watcherSets = $watcherSQL->runLoop(
+			wfGetDB( DB_SLAVE ),
+			function ( &$watcherSets, stdClass $row ) use ( &$currentPage, &$watcherCount ) {
+				$title = $row->wl_title;
+				$user = $row->wl_user;
+				$watcherSets[$currentPage][$title][] = $user;
+
+				$watcherCount++;
+				if ( ( $watcherCount % self::MAX_WATCHERS_PER_JOB ) == 0 ) {
```

Comment:
https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#Equality_operators
Triple-equals is preferred.